Tensile/SolutionStructs.py

```diff
@@ -2954,6 +2954,47 @@ def assignDerivedParameters(state):
         if state["LocalSplitU"] > 1:
           reject(state, "Atomic Stream-K not working with LocalSplitU")
 
+    def supportsPreloadKernelArguments():
+      if not globalParameters["AsmCaps"][isa]["KernargPreloading"]:
+        printWarning(f"{isa} doesn't support preloading.")
+        return False
+      
+      if not state["ProblemType"]["StridedBatched"]:
+        printWarning("Preloading only supported in StridedBatched mode.")
+        return False
+
+      dt = state["ProblemType"]["DataType"]
+      if not (dt.isSingle() or dt.isHalf() or dt.isBFloat16):
+        printWarning(f"Preloading not supported for data type {dt}.")
+        return False
+
+      if not state["ProblemType"]["UseBeta"]:
+        printWarning("Preloading only supported with UseBeta.")
+        return False
+
+      if state["ProblemType"]["UseInitialStridesAB"]:
+        printWarning("Preloading not supported with initial strides AB.")
+        return False
+
+      if state["WorkGroupMapping"] != 0:
```

Comment:
Can we changed this to abs(state["WorkGroupMapping"])>1 to allow preloading for WGM=1?